manifests/exec.pp

```diff
@@ -34,10 +35,12 @@
     $sanitised_container = $container
   }
   $exec = "${docker_command} exec ${docker_exec_flags} ${sanitised_container} ${command}"
+  $unless_command = "${docker_command} exec ${docker_exec_flags} ${sanitised_container} ${unless}"
 
   exec { $exec:
     environment => 'HOME=/root',
     path        => ['/bin', '/usr/bin'],
     timeout     => 0,
+    unless      => $unless_command,
```

Comment:
I think this needs a guard clause? If unless isn't passed to the `docker::exec` it will be passed to the underlying exec, but with an incomplete `unless_command`.